cdap-data-fabric/src/main/java/co/cask/cdap/data2/datafabric/dataset/service/executor/DatasetAdminOpHTTPHandler.java

```diff
@@ -104,7 +105,7 @@ public void create(HttpRequest request, HttpResponder responder,
     }
 
     DatasetSpecification spec = type.configure(name, props);
-    DatasetAdmin admin = type.getAdmin(spec);
+    DatasetAdmin admin = type.getAdmin(new DatasetContext.Builder().setNamespaceId(namespaceId).build(), spec);
```

Comment:
No other reason but future-proofing. There may be other fields in ``DatasetContext`` in future - other things that we may want to pass to storage providers.